lib/galaxy/webapps/base/api.py

```diff
@@ -1,16 +1,14 @@
 from fastapi import (
     FastAPI,
     Request,
+    status,
 )
 from fastapi.exceptions import RequestValidationError
 from fastapi.responses import JSONResponse
+from starlette.middleware.base import BaseHTTPMiddleware
```

Comment:
```suggestion
from starlette.middleware.base import (
    BaseHTTPMiddleware,
    RequestResponseEndpoint,
)
```

Reply:
Thanks, rebase this into https://github.com/galaxyproject/galaxy/pull/14202/commits/d5415a48b8e30d1e92c22508be4d9c6f41ac5e08!